docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
+
+CumulusCI's standard flows that build orgs, such as ``dev_org`` and ``install_prod``, always deploy metadata bundles found in ``unpackaged/post``, making it a full-fledged part of the application. It's also easy to include ``unpackaged/post`` metadata in customer-facing installers run via MetaDeploy.
+
+The ``deploy_post`` task, which is part of the ``config_dev``, ``config_qa``, and ``config_managed`` flows, is responsible for deploying these bundles.
+
+.. important:: Do not include metadata in ``unpackaged/post`` unless it is intended to be delivered to *all* environments (both managed installations and unmanaged deployments). It's also critical for managed package projects that this metadata include namespace tokens (see `namespace injection`_).
+
+
+``unpackaged/config``: Tailor an Org
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Projects can come with more than one supported configuration in their CumulusCI automation.
+
+    Example: Projects often support distinct, tailored ``dev_org``, ``qa_org``, and ``install_prod`` flows, each of which performs a unique setup for their specific use case.
+
+Unpackaged metadata stored in ``unpackaged/config`` is a tool to support operational needs that tailor orgs to different configurations. 
+
+    Example: A testing-oriented scratch org needs to deploy a customized set of Page Layouts to help testers easily visualize data under test. Such page layouts are stored in ``unpackaged/config/qa``.
+
+
+
+Unpackaged Metadata Folder Structure
+------------------------------------
+
+All unpackaged metadata is stored in the ``unpackaged`` directory tree, which contains these top-level directories.
+
+* ``unpackaged/pre``
+* ``unpackaged/post``
+* ``unpackaged/config``
+
+These trees contain metadata bundles in Metadata API format, represented as a directory containing a ``package.xml`` manifest and Metadata API-format source code. CumulusCI does not support Salesforce DX format for unpackaged bundles.
+
+
+
+Namespace Injection
 -------------------
 
-Customize config flows
+Projects that build managed packages often construct their unpackaged metadata to be deployable in multiple contexts.
+
+* Unmanaged deployments, such as developer orgs
+* Unmanaged namespaced scratch orgs
+* Managed contexts, such as a beta test org or a demo org created with ``install_prod``
+
+Because projects that are building an org implementation or a non-namespaced package do not have a namespace, or a distinction between managed and unmanaged contexts, these projects must also build namespaced packages.
+
+    Example: Metadata located in ``unpackaged/post`` is deployed after the application code in both unmanaged and managed contexts. If that metadata contains references to the application components, it must be deployable when that metadata is nampespaced (in a managed context or namespaced scratch org) *and* when it is not (in an unmanaged context).
```

Comment:
This is getting hard to follow simply because I don’t fully know why things are required to be namespaced. Which is okay because I'm still quite the novice when it comes to building anything with ``cci``. That said, is it safe to presume that most devs this deep into the material know about namespaces, why they're needed and will follow this material with (relative) ease?

Reply:
This example is clear to me (not sure how helpful this is), but I think it should be safe to assume that people reading this are going to be interested in, or familiar with, Salesforce packages in general. We might want to change this example based on feedback from David R. regarding your prior question.

Reply:
I'm happy to expand on or clarify this example but I think it's an important one because it is very common.

Reply:
Let's keep this then.